src/components/grid-aware/OneParagraphBlock/OneParagraphBlock.tsx

```diff
@@ -42,8 +44,14 @@ const OneParagraphBlock = ({
   );
 
   return (
-    <div className={s.bleedWrapper}>
-      <div className={s.bleedBackground} />
+    <div
+      className={`${s.bleedWrapper} 
+      ${s[theme]}`}
```

Comment:
Is it possible to put this on a single line? It'll look much better if we can, but I don't know if Prettier has an issue with it.

Same goes for the other one below.